functions/fk.fish

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env fish
+
+function fk -d "Command line utility for faster development process" -a cmd
+    if test ! -e $HOME/.config/fish/completions/fk.fish
+        echo "fk complete" > $HOME/.config/fish/completions/fk.fish
+        _fk_complete
+    end
+
+    while test -z "$fk_github_user";
```

Comment:
This would be required every time the user opens a new terminal and does fk <something> even is he/she doesn't need to use clone/repo.

Can we take this as an input once and store it in a file 
eg: `~/.fkconfig`. Also might be applicable for the current zsh implementation.

Reply:
This is a [universal variable](https://fishshell.com/docs/current/tutorial.html#tut_universal).

It essentially means that once you set it, it's set for future sessions as well. You're right, we should store it in a file, and we do. Fish does the heavy lifting for us.